functions/src/custom/vanity.ts

```diff
@@ -3,6 +3,16 @@ import * as Sentry from "@sentry/node";
 import request from "request";
 import sendgrid from "@sendgrid/mail";
 
+interface vanity {
```

Comment:
```suggestion
interface Vanity {
```

I think using the standard casing convention for type names would help code glanceability.